src/Core/src/Layouts/ILayoutManager.cs

```diff
@@ -6,6 +6,7 @@ namespace Microsoft.Maui.Layouts
 	public interface ILayoutManager
 	{
 		Size Measure(double widthConstraint, double heightConstraint);
-		Size ArrangeChildren(Rectangle childBounds);
+
+		Size ArrangeChildren(Size finalSize);
```

Comment:
Just an FYI that this is an ABI break for P7 that may break layout implementors. So we may have to let them know before we release.